server/config/tables.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable no-console */
+import pool from './db';
+
+pool.on('connect', () => {
+  console.log('Connected to PSQL DB');
+});
+
+// Drop Tables
+const drop = () => {
+  const users = 'DROP TABLE IF EXISTS users CASCADE;';
+  const announcements = 'DROP TABLE IF EXISTS announcements CASCADE;';
+  const flags = 'DROP TABLE IF EXISTS flags CASCADE;';
+  const con = `${users}; ${announcements}; ${flags};`;
```

Comment:
Would you mind renaming these variables properly?? a name that represents what they store

Reply:
Noted.